model_compression_toolkit/trainable_infrastructure/keras/README.md

```diff
@@ -9,7 +9,7 @@ Using this base class makes it simple to implement new quantizers for training a
 The [`@mark_quantizer`](https://github.com/sony/mct_quantizers/blob/main/mct_quantizers/common/base_inferable_quantizer.py) decorator is used to supply each quantizer with static properties which define its task compatibility. Each quantizer class should be decorated with this decorator. It defines the following properties:
  - [`QuantizationTarget`](https://github.com/sony/mct_quantizers/blob/main/mct_quantizers/common/base_inferable_quantizer.py): An Enum that indicates whether the quantizer is designated for weights or activations quantization.
  - [`QuantizationMethod`](https://github.com/sony/mct_quantizers/blob/main/mct_quantizers/common/quant_info.py): A list of quantization methods (Uniform, Symmetric, etc.).
- - `quantizer_type`: An Enum that defines the type of the quantization technique (varies between different quantization tasks).
+ - `identifier`: A unique identifier for the quantizer class. This is a helper property that allows the creation of advanced quantizers for specific tasks.
 
```

Comment:
isn't it the training method?

Reply:
For QT it's the training method, for GPTQ it's the rounding method, and for other quantizers it now can be any unique identifier